apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md

```diff
@@ -0,0 +1,107 @@
+
+### Improvement and enhancement of functionalities
+
+Out of 55 CZI EOSS grants for Python software projects, 40% involve projects that list SciPy as a dependency, therefore, making improvements to SciPy would directly benefit all these projects.
+
+By surveying the biomedical community, we seek to better support their work directly by adding new features and addressing critical issues they are facing. Preparing for the proposal, the team began a literature review and sent a public survey to maintainers of Python packages depending on SciPy. This continuous effort helps the project maintainers to better understand this community’s needs, their current uses of SciPy, and improvements they expect from SciPy.
+
+An associated, and paramount, objective is to build strong and long-lasting relationships with the biomedical community. Everyone is welcome to participate in this effort by providing feedback: head over to https://scipy.org/ to reach out.
+
+### General maintenance
+
+For more than 20 years, SciPy has been a reliable and stable library, trusted by millions of daily users. This implies a constant monitoring of bug reports and a willingness to listen to feedback coming from all user communities. The task by itself is challenging, and being an open source project means that such vital work stays largely unfunded, thus relying on volunteer work.
+
+Only a handful of maintainers are willing to spend time on support tasks such as working on the continuous integration system, the documentation infrastructure or even issue triaging. Allocating some time to these tasks, will be a service to the volunteer maintainers, who will be able to dedicate more time to the heart of SciPy instead of support tasks.
+
+### Dissemination of results
+
+Although SciPy is well established in the scientific Python ecosystem, its broad scope makes it difficult for a user to notice all its functionalities. Examples of libraries duplicating some “hidden” features of SciPy are numerous. This is a general observation in Science, where things are sometimes forgotten for years, if not centuries. The Fast Fourier Transform was discovered by Gauss more than a century before Fourier, Cooley and Tukey re-discovered it.
+
+The dissemination of scientific knowledge and tools is vital to the open source mission and sustainability of projects like SciPy. To achieve this, the team is making time for ongoing outreach activities such as community calls, and is active on community platforms like Slack and Discord. In addition, the team is planning to promote SciPy’s features at conferences and write specific tutorials targeting Biomedical researchers.
+
+### Call for contributions
+
+SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Head over to https://scipy.org/ and help us shape SciPy for the next decade.
```

Comment:
```suggestion
SciPy is open source, and as such, this grant work will also be done in the open and with the broaders community. It means that you can all contribute to help or provide feedback. Join [our community](https://scipy.org/community/), and help us shape SciPy for the next decade.
```

Reply:
avoid repetition from previous feedback section where "head over" is already used.